app/src/main/java/org/ieeeguc/ieeeguc/controllers/LoginActivity.java

```diff
@@ -1,28 +1,210 @@
 package org.ieeeguc.ieeeguc.controllers;
 
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
 import android.os.Bundle;
+import android.support.design.widget.Snackbar;
 import android.support.v7.app.AppCompatActivity;
-
+import android.util.Log;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.widget.AutoCompleteTextView;
+import android.widget.Button;
+import android.widget.EditText;
+import org.ieeeguc.ieeeguc.HTTPResponse;
 import org.ieeeguc.ieeeguc.R;
+import org.ieeeguc.ieeeguc.models.User;
+import org.ieeeguc.ieeeguc.models.User.Gender;
+import org.ieeeguc.ieeeguc.models.User.Type;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import static org.ieeeguc.ieeeguc.models.User.login;
 
 /**
  * A login screen that offers login via email/password.
  */
 public class LoginActivity extends AppCompatActivity {
 
+    AutoCompleteTextView email ;
+    EditText password ;
+    Button send ;
+
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
-    }
+        email = (AutoCompleteTextView) findViewById(R.id.email) ;
+        password = (EditText) findViewById(R.id.password) ;
+        send = (Button) findViewById(R.id.email_sign_in_button) ;
+        final String[] message = new String[1];
+        send.setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(View view) {

```

Comment:
move the code of this method to an external method that is called `attemptLogin();` and call it here